pennylane/vqe/vqe.py

```diff
@@ -24,7 +24,21 @@
 
 
 class ExpvalCost:
-    """Create a cost function that gives the expectation value of an input Hamiltonian.
+    """
```

Comment:
@Qottmann In case you were building it from source, it might be useful to use the Read the Docs CI check at the bottom of all PRs:

![image](https://user-images.githubusercontent.com/2959003/168187739-5e5c8bf7-f37d-48bb-9f11-825a87b013c2.png)

If you click details, you will be taken to a page to see how it will render:

https://pennylane--2571.org.readthedocs.build/en/2571/code/api/pennylane.ExpvalCost.html